pkg/ingestor/parser/parser.go

```diff
@@ -44,6 +44,7 @@ func init() {
 	_ = RegisterDocumentParser(dsse.NewDSSEParser, processor.DocumentDSSE)
 	_ = RegisterDocumentParser(slsa.NewSLSAParser, processor.DocumentITE6SLSA)
 	_ = RegisterDocumentParser(vuln.NewVulnCertificationParser, processor.DocumentITE6Vul)
+	_ = RegisterDocumentParser(spdx.NewSpdxParser, processor.DocumentITE6SPDX)
```

Comment:
it can be expanded [here](https://github.com/guacsec/guac/blob/main/pkg/handler/processor/ite6/ite6.go) similar to how its done for [DSSE](https://github.com/guacsec/guac/blob/main/pkg/handler/processor/dsse/dsse.go#L55)